molgenis-core-ui/src/main/resources/js/component/SelectBox.js

```diff
@@ -0,0 +1,42 @@
+/* global _: false, React: false, molgenis: true */
+(function(_, React, molgenis) {
+	"use strict";
+	
+	var div = React.DOM.div;
+	var select = React.DOM.select;
+	var option = React.DOM.option;
+	
+	/**
+	 * @memberOf component
+	 */
+	var SelectBox = React.createClass({
+		mixins: [molgenis.ui.mixin.DeepPureRenderMixin],
+		displayName: 'SelectBox',
+		propTypes: {
+			options: React.PropTypes.array,
+			onChange: React.PropTypes.func
+		},
+		getDefaultProps: function(){
+			return {
+				options: [{value: "val1", text: "option1"}]
+			}
+		},
+		render: function() {
+			return (
+				div(null,
+					select({className: "form-control", onChange: this.props.onChange}, this.props.options.map(function(item, i){
+						return(
+							option({value: item.value, key: i}, item.text)
```

Comment:
I have a vague recollection that the value of key should be a string. If that is the case then i.toString().